tendrl/gluster_integration/message/callback.py

```diff
@@ -676,6 +677,86 @@ def brick_replace(self, event):
         event["message"] = brick_details
         self.volume_remove_brick_force(event)
 
+    def snapshot_restore(self, event):
+        message = event["message"]
+        volume = message['Volume']
+        volume_id = ""
+        bricks_to_remove = []
+
+        # get the list of current bricks by running get-state
+        subprocess.call(
+            [
+                'gluster',
+                'get-state',
+                'glusterd',
+                'odir',
+                '/var/run',
+                'file',
+                'glusterd-state-snapshot',
```

Comment:
this file name needs to be unique per event, please add a uuid of something

Reply:
ack